src/collaborative_platform/apps/close_reading/templates/close_reading/close_reading.html

```diff
@@ -125,7 +125,7 @@ <h3 class="break-word">
                                   <label for="category">Category of uncertainty 
                                     <span class="help-tooltip" help="This is the category." />
                                   </label>
-                                  <select class="form-control form-control-sm" id='category'>
+                                  <select multiple class="form-control form-control-sm" id='category'>
                                     <option class="ignorance" value="ignorance">Ignorance</option>
                                     <option class="credibility" value="credibility">Credibility</option>
                                     <option class="imprecision" value="imprecision">Imprecision</option>
```

Comment:
@virdiignis Is this what seems to be hardcoding?

Reply:
Yes. When we spoke about elastic taxonomy ealier, we agreed that the taxonomy catagories should be passed by django template, and used in annotator in place of those hardcoded values.

Reply:
> Yes. When we spoke about elastic taxonomy ealier, we agreed that the taxonomy catagories should be passed by django template, and used in annotator in place of those hardcoded values.

Ok, thanks yeah that's true. Tomorrow and Monday are bank holidays here but @Janchorizo will look into this asap.

Reply:
I indeed missed that HTML form. I have pushed the changes to render the options accordingly
to the specified taxonomy. 
Apart from that the legend was already handled properly, and there is a separate JSON
with a default scheme for the entities just in case we decide to change it, or allow the
user to change it.